external/src/main/java/com/redhat/cloud/policies/engine/AlertStarter.java

```diff
@@ -49,6 +52,7 @@ public ActionsService getActionsService() {
     }
 
     void startApp(@Observes StartupEvent startup) {
+        printEnvVars();
```

Comment:
Are we sure this is early enough? I don't recall if config resolution happens earlier or not.
And then it may be no issue on engine, as the values are not injected, but obtained by running code.

Reply:
The current prod failure happens after that so it should be ok.